### PR TITLE
[NO TICKET] Route failures to Workspaces test alert channel.

### PR DESCRIPTION
**Pull request by cahrens**
Move notifications from #terra-wsm-alerts to the Workspaces team channel. The older Slack channel will be archived.

### CODE REVIEW

File: .github/workflows/full-integration-tests.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       with:
         status: ${{ job.status }}
-        channel: "#terra-wsm-alerts"
+        channel: "#dsp-workspace-test-alerts"
```
Comment (cahrens) [inline .github/workflows/full-integration-tests.yml:95]: Notifies on failure.

File: .github/workflows/full-service-tests.yml [hunk 2]
```diff
@@ -120,7 +120,7 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       with:
         status: ${{ job.status }}
-        channel: "#terra-wsm-alerts"
+        channel: "#dsp-workspace-test-alerts"
```
Comment (cahrens) [inline .github/workflows/full-service-tests.yml:123]: Notifies on failure. Will also notify to #dsde-qa always for alpha and staging.

File: .github/workflows/release-promotion-tests.yml [hunk 3]
```diff
@@ -175,6 +175,19 @@ jobs:
           author_name: "Workspace Manager ${{ steps.set-env-step.outputs.test-env }} integrationTest"
           fields: repo,job,workflow,commit,eventName,author,took
 
+      - name: "Notify Workspaces test failure Slack"
+        if: failure()
```
Comment (cahrens) [inline .github/workflows/release-promotion-tests.yml:179]: Add to notify on failure to our channel. There is a previous step to notify to #dsde-qa on promotion to alpha and staging.

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -217,7 +217,7 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       with:
         status: ${{ job.status }}
-        channel: "#terra-wsm-alerts"
+        channel: "#dsp-workspace-test-alerts"
```
Comment (cahrens) [inline .github/workflows/test.yml:220]: Notify on failure, non-bumper push. Also notifies in previous step to #dsde-qa always if alpha/staging.

File: .github/workflows/full-integration-tests.yml (hunk 1 above)
Comment (aherbst-broad) [inline .github/workflows/full-integration-tests.yml:95]: I think this is #dsp-workspaces-test-alerts (note the plural)

Comment (cahrens) [reply .github/workflows/full-integration-tests.yml:95]: Ah, good catch! I copied this from a BPM github action, so I'll update it there also.

**Review by sergiygetlin — APPROVED** [review]

**Review by aherbst-broad — APPROVED** [review]

**Pull request merged**